project/ws/app/src/lib/routes/home/routes/designation/components/designations/designations.component.html

```diff
@@ -1,7 +1,10 @@
 <div class="flex flex-column gap-4 pt-4">
   <div class="flex justify-between items-center">
     <div class="mobile-margin text-capitalize mat-h2">{{designationConfig.mainHeading}}</div>
-    <div>
+    <div class="flex gap-3">
+      <button mat-raised-button class="importBtn" [routerLink]="'bulk-upload'" type="button">
+        Bulk Upload Designation
```

Comment:
in line 3 you can see main heading is being read from json file, do similar thing for this